server/db/models/player.js

```diff
@@ -17,6 +17,7 @@ const Player = db.define('player', {
   username: {
     type: Sequelize.STRING,
     allowNull: false,
+    defaultValue: 'defaultName',
```

Comment:
I think defaultName is fine for now, but in the future we may want just a "placeholder" text or something along the lines but that can be later, and I think for now its fine! maybe we can randomly generate a username if a user doesn't put one in? just some thoughts!

Reply:
I agree - there was a reason why I did this, it had something to do with the input field's default value for the username form modal on front end didn't like it if I didn't have a default Value.
On second thought, maybe we can give it default of an empty string.